Quadriga/src/main/java/edu/asu/spring/quadriga/web/transformation/UploadTransformFilesController.java

```diff
@@ -81,11 +84,25 @@ public ModelAndView uploadTransformFiles(
             String mappingDescription = formBean.getMappingDescription();
             String mappingFileName = file[1].getOriginalFilename();
 
-            transformationManager.saveTransformation(title, description,
-                    patternFileName, patternTitle, patternDescription,
-                    mappingFileName, mappingTitle, mappingDescription,userName);
-            // Only meta data is being saved in database. Saving files is not
-            // yet done..
+            TransformationFile transformationFile = new TransformationFile();
+            transformationFile.setTitle(title);
+            transformationFile.setDescription(description);
+            transformationFile.setUserName(userName);
+            transformationFile.setPatternTitle(patternTitle);
+            transformationFile.setPatternDescription(patternDescription);
+            transformationFile.setPatternFileName(patternFileName);
+            transformationFile.setPatternFileContent(file[0].getBytes());
+            transformationFile.setMappingTitle(mappingTitle);
+            transformationFile.setMappingDescription(mappingDescription);
+            transformationFile.setMappingFileName(mappingFileName);
+            transformationFile.setMappingFileContent(file[1].getBytes());
+
+            try {
+                transformationManager.saveTransformations(transformationFile);
+            } catch (FileStorageException e) {
+                logger.error("Error in UploadTransformFiles Controller:", e);
+                throw new FileStorageException();
```

Comment:
don't swallow the exception and don't log it. It'll be logged by whoever catches it.